src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -474,22 +475,22 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         state->model_name = std::move(*model_name_string);
 
         // Pull the model repository path out of the configuration.
-        auto model_repo_path = attributes->find("model_repository_path");
-        if (model_repo_path == attributes->end()) {
+        auto model_repo_path = attributes.find("model_repository_path");
+        if (model_repo_path == attributes.end()) {
```

Comment:
If it compiles, I can't argue with it. but I missed where `attributes` has changed types from a pointer to an object, to switch from using `->` to using `.`. Where did that happen?

Reply:
It didn't change in this review, it changed in the C++ SDK from `AttributeMap` to `ProtoStruct`: https://github.com/viamrobotics/viam-cpp-sdk/commit/90246f012c56566cb70ed84cdc36d4d574080ba9#diff-948b97110a0cae9657a8ee872763c73ce056e898d9aa26367c56ef3c31cfd363